src/Parser/RetributionPaladin/Modules/Items/Tier20_2set.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const RET_PALADIN_T20_2SET_MODIFIER = .2;
+
+class Tier20_2set extends Module {
+	static dependencies = {
+		combatants: Combatants,
+	};
+
+	damageDone = 0;
+
+	on_byPlayer_damage(event) {
+		if(this.combatants.selected.hasBuff(SPELLS.RET_PALADIN_T20_2SET_BONUS_BUFF.id)){
```

Comment:
The recommended style is to invert these if statements and return early instead of nesting blocks. Keeps things more readable.